edx_username_changer/__init__.py

```diff
@@ -0,0 +1,2 @@
+__version__ = "0.0.1"
```

Comment:
Although the [draft](https://github.com/edx/open-edx-proposals/pull/181/files) says that this file should only have `default_app_config` in it but I believe the intention of saying that was to restrict exporting methods in this file.

In case we want to remove the `__version__` variable from this file then we might need to place it in some kind of `constant.py` and edit our doof accordingly.

Reply:
@HamzaIbnFarooq In case of plugins version should be in `setup.py` of plugin.

Reply:
version is declared here (due to some deployment-related limitations) but being used in setup.py